HISTORY.md

```diff
@@ -30,7 +30,7 @@
 * Rename `SizeApproximationOptions.include_memtabtles` to `SizeApproximationOptions.include_memtables`.
 * Remove deprecated option DBOptions::max_mem_compaction_level.
 * Remove deprecated overloads of API DB::GetApproximateSizes.
-
+* Add DB::OpenAndTrimHistory API. This API will open DB and trim data to the timestamp specified by trim_ts (The data with newer timestamp than specified trim bound will be removed). This API should only be used at a timestamp-enabled db instance recovery.
```

Comment:
> a timestamp-enabled db instance

Actually timestamp is a per-cf setting, not db.

Have you considered the case of database in which some column families enable timestamp while others do not?

Reply:
Yes, you are right. It is reasonable to skip the column family without timestamp enabled when OpenAndTrimHistory performs the compaction operation.